src/Framework/MSBuildEventSource.cs

```diff
@@ -551,6 +551,54 @@ public void OutOfProcSdkResolverServiceRequestSdkPathFromMainNodeStop(int submis
             WriteEvent(75, submissionId, sdkName, solutionPath, projectPath, success, wasResultCached);
         }
 
+        [Event(76, Keywords = Keywords.All | Keywords.PerformanceLog)]
```

Comment:
Please remove `PerformanceLog` from all of these except maybe CacheGet Start/Stop. I'll send up a PR to add doc comments explaining why.

Reply:
dotnet/msbuild#7307 adds explanation